lib/ama_layout/notification_set.rb

```diff
@@ -20,7 +21,7 @@ class NotificationSet
     include Enumerable
     attr_accessor :base, :data_store, :key
 
-    delegate :each, :first, :last, :size, :[], :empty?, :any?,  to: :active
+    delegate :each, :first, :last, :size, :[], :empty?, :any?,  to: :all
```

Comment:
We spoke about this in person. As of right now the delegation is needed as many tests are written assuming a `NotificationSet` responds to array-like methods.